src/WebDriver/WebDriverMouse.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+/*
+ * This file is part of the Panther project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Symfony\Component\Panther\WebDriver;
+
+use Facebook\WebDriver\Interactions\Internal\WebDriverCoordinates;
+use Facebook\WebDriver\Internal\WebDriverLocatable;
+use Facebook\WebDriver\WebDriverMouse as BaseWebDriverMouse;
+use Symfony\Component\Panther\Client;
+
+/**
+ * @author Dany Maillard <danymaillard93b@gmail.com>
+ */
+class WebDriverMouse implements BaseWebDriverMouse
+{
+    /**
+     * @var BaseWebDriverMouse
+     */
+    private $mouse;
+
+    /**
+     * @var Client
+     */
+    private $client;
+
+    public function __construct(BaseWebDriverMouse $mouse, Client $client)
+    {
+        $this->mouse = $mouse;
+        $this->client = $client;
+    }
+
+    public function click(WebDriverCoordinates $where)
```

Comment:
Can you add the return types to all methods?

Reply:
Done.